packages/block-editor/src/components/url-input/index.js

```diff
@@ -235,7 +235,8 @@ class URLInput extends Component {
 	}
 
 	onChange( event ) {
-		this.props.onChange( event.target.value );
+		this.props.onChange( event );
```

Comment:
To make the code clearer to understand, we could change `event` to `newValue`

Reply:
Yes, newValue sounds good